allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -9,6 +9,8 @@
 
 logger = logging.getLogger(__name__)
 
+ALLENNLP_SENTENCE_PAIR_SEP = "@@SEP_SENTPAIR@@"
```

Comment:
As I said in another comment, I'm not thrilled with having this.  But if we do want it, it should not belong here, it should be in something like `data/common.py`.  And I would name it `SENTENCE_SEPARATOR`, instead of what you have.